languages/python3-adk/template/algorithmia.conf

```diff
@@ -0,0 +1,5 @@
+{
+    "username": "__USER__",
+    "algoname": "__ALGO__",
+    "langauge": "python3.7"
```

Comment:
I'm not clear in how we use these .conf files, but in case they're functional and we want to keep them around; we can fix the typo of `langauge`->`language`

Reply:
yeah good point, this might be a good question for @algorithmiaio/product-services as I know the concept of `language` is defined in many locations. I believe here it's just user facing information, but that might be worth adjusting

Reply:
good news, that language field is actually just used in the template repo now, it's completely detatched from the rest of the system, so we can set that to be `python3.x-adk` or.. anything really